pkg/ansible/run.go

```diff
@@ -275,18 +275,25 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string, gvks []schema.GroupVers
 // getAnsibleDebugLog return the value from the ANSIBLE_DEBUG_LOGS it order to
 // print the full Ansible logs
 func getAnsibleDebugLog() bool {
-	const envVar = "ANSIBLE_DEBUG_LOGS"
+	envVars := []string{"ANSIBLE_DEBUG_LOGS", "ANSIBLE_ENABLE_TASK_DEBUGGER"}
 	val := false
-	if envVal, ok := os.LookupEnv(envVar); ok {
```

Comment:
The only change here is that if the `ANSIBLE_ENABLE_TASK_DEBUGGER` be set than the Ansible logs will appear in the operator logs. Is that what you are trying to achieve?

Reply:
Double checked the `ANSIBLE_ENABLE_TASK_DEBUGGER` doc [here](https://docs.ansible.com/ansible/latest/user_guide/playbooks_debugger.html) then I don't think this is the right way to enable task debugger. The `ANSIBLE_ENABLE_TASK_DEBUGGER` env var must be communicated to the `ansible-playbook` command line, while this PR obviously is not doing that.